### PR TITLE
Define schema for updateCustomerAccountCard

### PR DESCRIPTION
**Pull request by kpamaran**
## Description of the change

For https://canopyservicing.atlassian.net/browse/PCV1-39

## Type of change

- [ ] Bug fix (non-breaking change that fixes an issue)
- [x] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

### Development

- [x] Lint rules pass locally
- [ ] The code changed/added as part of this pull request has been covered with tests
- [ ] All tests related to the changed code pass in development

### Code review

- [x] This pull request has a descriptive title and information useful to a reviewer. There may be a screenshot or screencast attached
- [ ] "Ready for review" label attached to the PR and reviewers mentioned in a comment
- [ ] Changes have been reviewed by at least one other engineer
- [ ] Issue from task tracker has a link to this pull request

### CODE REVIEW

File: dist/consolidated.json [hunk 1]
```diff
@@ -7871,6 +7871,148 @@
...
+            "description": "Customer ID",
+            "required": true,
+            "style": "simple",
+            "explode": false,
+            "schema": {
+              "type": "string",
+              "example": "jNxDp2ybR5M72LaBA83e06md"
+            }
+          }
+        ],
```
Comment (ps2-controller) [inline dist/consolidated.json:7907]: Why is card_id not needed? Can a single customer_account have multiple cards? I forget

**Review by ps2-controller — CHANGES_REQUESTED** [review]

File: dist/consolidated.json (hunk 1 above)
Comment (kpamaran) [reply dist/consolidated.json:7907]: Yes there can be multiple cards per customer

Comment (ps2-controller) [reply dist/consolidated.json:7907]: So how do we know which card is being modified by this request?

File: schemas/UpdateCustomerAccountCardInput.json [hunk 2]
```diff
@@ -0,0 +1,26 @@
+{
+  "type": "object",
+  "properties": {
+    "lithic": {
+      "type": "object",
+      "properties": {
+        "card_token": {
```
Comment (kpamaran) [inline schemas/UpdateCustomerAccountCardInput.json:7]: @ps2-controller this is the identifier for a Lithic card.

Comment (ps2-controller) [reply schemas/UpdateCustomerAccountCardInput.json:7]: ahhh right awesome!

**Review by ps2-controller — APPROVED** [review]

**Comment by kpamaran** [comment]
Will merge once API branch is ready

**Comment by ps2-controller** [comment]
Sounds good, thanks for converting to a draft til then so it doesn't get merged mistakenly

**Pull request merged**